### ISSUE TITLE
Touching graphical applications: bimanual tactile interaction on the HyperBraille pin-matrix display

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#328]
### Links
- Paper: https://link.springer.com/article/10.1007/s10209-017-0538-8

<img width="506" alt="ScreenShot 2020-05-30 22 57 25" src="https://user-images.githubusercontent.com/26996041/83330294-fd985800-a2c8-11ea-9c35-443341766587.png">
<img width="550" alt="ScreenShot 2020-05-30 22 57 14" src="https://user-images.githubusercontent.com/26996041/83330296-fec98500-a2c8-11ea-95ba-971268f04ff3.png">

### Abstract
Novel two-dimensional tactile displays enable blind users to not only get access to the textual but also to the graphical content of a graphical user interface. Due to the higher amount of information that can be presented in parallel, orientation and exploration can be more complex. In this paper we present the HyperBraille system, which consists of a pin-matrix device as well as a graphical screen reader providing the user with appropriate presentation and interaction possibilities. To allow for a detailed analysis of bimanual interaction strategies on a pin-matrix device, we conducted two user studies with a total of 12 blind people. The task was to fill in .pdf forms on the pin-matrix device by using different input methods, namely gestures, built-in hardware buttons as well as a conventional PC keyboard. The forms were presented in a semigraphic view type that not only contains Braille but also tactile widgets in a spatial arrangement. While completion time and error rate partly depended on the chosen input method, the usage of special reading strategies seemed to be independent of it. A direct comparison of the system and a conventional assistive technology (screen reader with single-line Braille device) showed that interaction on the pin-matrix device can be very efficient if the user is trained. The two-dimensional output can improve access to .pdf forms with insufficient accessibility as the mapping of input controls and the corresponding labels can be supported by a spatial presentation.

- GUIを扱えるように支援する触覚ディスプレイを提案
